### DISCUSSION TITLE
Help required In Building signal-server 7.x.x

### DISCUSSION THREAD

**Discussion by erick2441** [indrawp/Setup-Guide#144] [General]
Has anyone been successfull in building signal server 7.x.x? 
I've tried and I'm stuck at running the textsecureserver jar file. 

To  build 7.x.x, you will be required to have open-jdk-17 and maven 3.8.4.  
You can download maven 3.8.4 from here https://maven.apache.org/download.cgi 

Does anyone have any possible ideas on how to proceed?

**Comment by JonnyX11** [comment]
Hey,

Did you success to run the server 5.XX ? what about client iOS and android ?

If you can you please guide me ?

Thanks!

**Comment by naive-guy** [comment]
hi @erick2441, were you able to figure out how to run this jar?

**Comment by erick2441** [comment]
Nope
